includes/utils.php

```diff
@@ -199,6 +199,9 @@ function set_meta( $post_id, $meta ) {
 				$meta_value = maybe_unserialize( $meta_value );
 			}
 
+			// Adds slashes to a string or recursively adds slashes to strings within an array to avoid JSON decode failure while migrating metadata
+			$meta_value = wp_slash( $meta_value );
```

Comment:
Both the meta key and the meta value need to be slashed. 

To be more defensive with the code, I suggest slashing the data within the function calls, ie

```php
update_post_meta( $post_id, wp_slash( $meta_key ), wp_slash( $meta_value ), $prev_value );
// and
add_post_meta( $post_id, wp_slash( $meta_key ), wp_slash( $meta_value ) );
```

Although slashing the data at the moment is safe, escaping later protects from future mistakes if data that requires slashing is added between here and the calls to the core functions.

Reply:
Thank you so much for your feedback @peterwilsoncc, I've updated the PR